x11-misc/piedock/piedock-1.6.6-r1.ebuild

```diff
@@ -1,21 +1,21 @@
-# Copyright 1999-2016 Gentoo Foundation
+# Copyright 1999-2017 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 # $Id$
 
-EAPI=5
-inherit eutils
+EAPI=6
 
 DESCRIPTION="A little bit like the famous OS X dock but in shape of a pie menu"
 HOMEPAGE="http://markusfisch.de/PieDock"
-SRC_URI="http://markusfisch.de/downloads/${P}.tar.bz2"
+SRC_URI="http://markusfisch.de/downloads/${P}.tar.bz2
+	https://github.com/markusfisch/PieDock/commit/a7fda1896f1cc6966ba0fa8912e9b404c1b0be97.patch -> ${P}-gcc6.patch"
 
 LICENSE="MIT"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
 IUSE="gtk kde"
 
 RDEPEND="
-	media-libs/libpng
+	media-libs/libpng:0
```

Comment:
does it link to libpng (which I strongly would think)? linked libpng deps always require `:0=`